### PR TITLE
add guinea subsite

### PR DESCRIPTION
**Pull request by manishgs**

### CODE REVIEW

**Review by iprunache — COMMENTED** [review]
@manishgs this looks good!

The build is failing though because of changes to the alpine image we use as base in the Dockerfile. Could you try with `FROM alpine:3.4` instead of latest and see if the build still fails?

**Comment by iprunache** [comment]
Trying to find out which was the latest version of `alpine` we used in a successful build.

**Comment by iprunache** [comment]
@manishgs I've changed the version of `alpine` to `3.5` myself which is what we run in production.

File: config/init.sh [hunk 1]
```diff
@@ -68,5 +67,19 @@ envsubst '$SERVER_NAME $CATEGORY'< ./nginx_subsite.template > /etc/nginx/sites-e
 export CATEGORY=olc
 envsubst < ./env.template > /var/www/country-tz/.env
 
+#country-guinea
+export SERVER_NAME="guinea.resourcecontracts.org *.guinea.resourcecontracts.org"
+export TRACKING_ID=${GN_TRACKING_ID}
+export DB_DATABASE=${COUNTRY_DB_DATABASE}
+export CONTACT_MAIL=${GN_CONTACT_MAIL}
+export ELASTIC_SEARCH_HOST=${GN_ELASTIC_SEARCH_HOST}
```
Comment (iprunache) [inline config/init.sh:75]: @manishgs where should we point this to? index.resourcecontracts.org?

Comment (manishgs) [reply config/init.sh:75]: Use production API (index.resourcecontracts.org) for guinea subsite on both environment.

**Pull request merged**